packages/remix-architect/package.json

```diff
@@ -17,7 +17,7 @@
     "@remix-run/server-runtime": "1.1.0"
   },
   "peerDependencies": {
-    "@architect/architect": "^8.3.7"
+    "@architect/architect": "^9.3.1"
```

Comment:
Please back out the version update on this PR and we'll get the functionality merged.

Reply:
As the commit message says (63590355ec19c6aa42381077fc5750cf918a2cf5):

> Bump architect version to 9.3.1
>
> This version adds a dependency on @architect/sandbox 4.3.1, which added the `ARC_SANDBOX` environment variable.

I found that `"@architect/architect": "^9.3.1"` is the earliest version that pull in @architect/sandbox 4.3.1.